README.md

```diff
@@ -1,58 +1,173 @@
+
+**Notes:**
+
+- *Sane & powerful template engine*
+  - hugo gets ![ehh] because golang templates are simply not meant for webdesign (despite being very powerful), to the point of driving @Keats insane to create his own template engine ([tera](https://github.com/keats/tera)) and static-site generator (zola) ; yes this is a little biased
+- *Relative URLs*
+  - zola gets ![ehh] because it can only produce relative URLs for the webroot, not a subfolder (see discussion about [path unification](https://github.com/getzola/zola/issues/711))
+
+### Deployment integrations
+
+While it's usually possible to use any kind of static site generator to deploy to a website, some platforms have restrictions and specific tooling to publish content. Here's some example documentation for deploying on specific platforms:
+
+|                                     | [Zola](https://getzola.org/)   | [Cobalt](https://cobalt-org.github.io/) | [Hugo](https://gohugo.io/)   | [Pelican](https://blog.getpelican.com/) |
+|:------------------------------------|:------:|:------:|:------:|:-------:|
+| [Netlify](https://www.netlify.com/) |  [![yes]](https://www.getzola.org/documentation/deployment/netlify/)  |  ![no]   |  ![yes]  |   ![no]  |
+| [Vercel](https://vercel.com/)       |  [![yes]](https://www.getzola.org/documentation/deployment/vercel/)  |  ![no]   |  ![yes]  |  ![yes]  |
+| [SourceHut](https://srht.site/)    |  [![yes]](https://www.getzola.org/documentation/deployment/sourcehut/)  |    ?     |     ?    |     ?    |
+| [Github](https://pages.github.com/) |  [![yes]](https://www.getzola.org/documentation/deployment/github-pages/) |   ?   | ![yes] | ![yes] |
+| [Gitlab](https://docs.gitlab.com/ee/user/project/pages/) | [![yes]](https://www.getzola.org/documentation/deployment/gitlab-pages/) | ? | ![yes] | ![yes] |
+
+## Contributing
+
+If you would like to contribute to improve zola together as a community, there are many ways to do so. In this section, we outline a few ways you could help the zola project. If you're looking to make a contribution now, please read [CONTRIBUTING.md](CONTRIBUTING.md).
+
+### Documentation and pro tips
+
+When you notice something isn't clear in the documentation, you may submit a patch to improve it for every one else. If you're doing something which you think could be of interest to other folks, please write about it [on the forums](https://zola.discourse.group/). If you notice some usage patterns emerging on the forums, feel free to turn that into a documentation page to help other people achieve the same goals.
+
+### Usability and architecture
 
-### Supported content formats
+Despite being three decades old, the world wide web isn't always the most convenient platform to use. zola tries to make your life better, but if you're experiencing difficulties with it, don't hesitate to report those. Please be clear about what was confusing or unexpected regarding what you tried to achieve.
```

Comment:
I think this is kind of a duplicate of a paragraph above.

Reply:
I don't think so? Documentation said about interesting hacks. This "Areas to improve" paragraph is about what is unexpected/inconsistent. But i can remove the paragraph if you don't like it.

Reply:
I don't think so? Documentation said about interesting hacks. This "Areas to improve" paragraph is about what is unexpected/inconsistent. But i can remove the paragraph if you don't like it.